src/Moq.EntityFrameworkCore.Examples/Moq.EntityFrameworkCore.Examples.csproj

```diff
@@ -1,18 +1,19 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="AutoFixture" Version="4.11.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.2" />
-    <PackageReference Include="Moq" Version="4.13.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="3.1.8" />
```

Comment:
I would leave the previous version of libs then it will accessible for the wider group of people.

Reply:
@MichalJankowskii I'm backing out these references. Just wanted to make sure you're aware though that when you compile this in current SDKs, this warning is shown:

warning NETSDK1138: The target framework 'netcoreapp3.0' is out of support and will not receive security updates in the future. Please refer to https://aka.ms/dotnet-core-support for more information about the support policy.